Python/AudioBook/Audiobook.py

```diff
@@ -0,0 +1,12 @@
+import pyttsx3
+import PyPDF2
+book=open('software.pdf','rb')
+pdfReader=PyPDF2.PdfFileReader(book)

```

Comment:
The Formatting is quite poor. I would suggest you to use a tool like PyLint to see how you can improve the Code Quality.